src/components/ViewResponses/Responses.js

```diff
@@ -0,0 +1,43 @@
+import { Card, Typography } from "antd";
+
+const Response = (props) => {
+  let { allResponses, userInfo } = props;
+
+  const { Title } = Typography;
+  allResponses && allResponses.length !== 0 && (
+    <Title level={3}> Responses</Title>
```

Comment:
not an urgent issue but we can create a map/constant file of all strings used. Will be easy later on if we want to add internationalisation

Reply:
will pick this up after this